react-validated-input/my-app/src/ValidatedInput.tsx

```diff
@@ -0,0 +1,41 @@
+import { useState } from 'react';
+import './ValidatedInput.css';
+import { IoIosCheckmark } from 'react-icons/io';
+import { TbLetterX } from 'react-icons/tb';
+
+export function ValidatedInput() {
+  const [value, setValue] = useState('');
+  function validationCheck() {
+    let validation = '';
+    if (value.length === 0) {
+      validation = 'A password is required.';
+    } else if (value.length < 8) {
+      validation = 'Your password is too short';
+    }
+    return validation;
+  }
+  function errorIcon() {
+    if (value.length >= 8)
```

Comment:
You can move the error variable from line 23 to above the errorIcon function.
Then use the length of the error variable in the if statement check for the errorIcon function.
This way, if you decide to add additional validation rules this function will still work  :)